template/function/serverless.yml

```diff
@@ -1,21 +1,29 @@
-service: app
+service: TPL_SERVICE_NAME
 
 provider:
-    name: aws
-    region: us-east-1
-    runtime: provided.al2
+  # general settings
+  name: aws
```

Comment:
4 spaces indentation plz (easier to read, less likely to make a mistake, complies with most PHP practices)